src/data/mainNavigationData.js

```diff
@@ -0,0 +1,43 @@
+import home from '../img/homeSolid.svg'
+import paperplane from '../img/paper-planeSolid.svg'
+import cog from '../img/cogSolid.svg'
+import swap from '../img/syncSolid.svg'
+import powerOff from '../img/power-offSolid.svg'
+
+export default [
```

Comment:
So, is there a was to get to the other routes? I think while we're still building out the new design we need to be able to get to create wallet etc.

Reply:
Yes, I will implement all the current development routes.